build-presets/MakeBOSTree.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import numpy as np
+from scipy.optimize import curve_fit
+import json
+
+efficiencies = []
+
+# 10% to 40% efficiency
+# building a list of efficiencies
+start = 0.1
+while (start < 0.41):
+	efficiencies.append(round(start, 2)) 
+	start += 0.01
```

Comment:
I'd suggest this more concise way to build the array. If a numpy array won't work with the curve_fit call below, can always change to a `list` first
```suggestion
efficiencies = np.arange(0.1,0.4,0.01)
```